### PR TITLE
chore: Build and package updates to allow use of webpack3

### LINKED ISSUE #471: Add webpack 3 support
Webpack 3 rc has been release. It would be great to make sure the loader works as expected with it.

### PR DESCRIPTION
**Pull request by d3viant0ne**
- Updates peerDependencies to allow use with Webpack 3.x
- Adds defaults travis & appveyor builds

Closes #471

### CODE REVIEW

**Comment by evilebottnawi** [comment]
@d3viant0ne Can we release path version today, after merge this?

**Comment by d3viant0ne** [comment]
That's the idea, just tidying up the package quick

File: package.json [hunk 1]
```diff
@@ -55,13 +44,24 @@
...
+  "keywords": [
+    "sass",
+    "libsass",
+    "webpack",
+    "loader"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/webpack-contrib/sass-loader.git"
+  }
```
Comment (michael-ciniawsky) [inline package.json]:
```
"bugs": {
  "url": "https://github.com/webpack-contrib/sass-loader/issues"
 },
 "homepage": "https://github.com/webpack-contrib/sass-loader",
```

File: package.json [hunk 2]
```diff
@@ -60,8 +60,8 @@
     "webpack",
     "loader"
   ],
-  "repository": {
```
Comment (michael-ciniawsky) [inline package.json]: This can still stay :)

**Pull request merged**